vm/gce/gce.go

```diff
@@ -138,6 +139,31 @@ func Ctor(env *vmimpl.Env, consoleReadCmd string) (*Pool, error) {
 	return pool, nil
 }
 
+const (
+	gceInitAttempts = 3
```

Comment:
Move these to the function, they are not used outside of it.

Reply:
Done.